doc/cli/vast-dump.md

```diff
@@ -0,0 +1,9 @@
+The `dump` command prints configuration options as JSON. An optional `--yaml`
+flag switches the output format to YAML.
+
+For example, to see the amalgamated concept definitions, use the following
+command:
+
+```
+vast dump --yaml concepts
+```
```

Comment:
You could add a note/tip that the YAML output is also valid input when placing it into a file in the loaded schema dirs.

Reply:
Do we want to make that guarantee? We're not testing it currently, and there are no checks in place for it.